modules/openapi-generator/src/main/resources/python/rest.mustache

```diff
@@ -61,56 +61,45 @@ class RESTClientObject:
         else:
             cert_reqs = ssl.CERT_NONE
 
-        addition_pool_args = {}
+        pool_args = {
+            "cert_reqs": cert_reqs,
+            "ca_certs": configuration.ssl_ca_cert,
+            "cert_file": configuration.cert_file,
+            "key_file": configuration.key_file,
+        }
```

Comment:
I simplified a bit the pool configuration:

* The pool object is correctly typed now
* The duplicated parameters are now all set here before hand
* Specific pools only add their own parameters.